kolibri/plugins/learn/assets/src/views/learn-page/index.vue

```diff
@@ -49,7 +49,7 @@
       popularSectionSubHeader: '{numOfItems, number} popular items',
       suggestedNextStepsSectionSubHeader: '{numOfItems, number} suggested items',
       resumeSectionSubHeader: '{numOfItems, number} items to be resumed',
-      overviewSectionHeader: 'Overview',
+      featuredSection: 'Featured in {nameOfChannel}',
```

Comment:
I would rather not rename this until we have done the multi-channel integration (and also because, string freeze).